yscope-compose/README.md

```
@@ -0,0 +1,76 @@
+After all containers are in "Started" states (check by `docker ps`):
+
+```bash
+# On your host
+docker exec -it compose-presto-coordinator-1 sh
+
+# In presto-coordinator container
+/opt/presto-cli --catalog clp --schema default --server localhost:8080
+```
+
+Example query:
+```sql
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)